src/ontology/fbbt-edit.obo

```diff
@@ -134736,7 +134738,7 @@ property_value: http://purl.org/dc/terms/date 2021-05-05T08:29:28Z xsd:dateTime
 [Term]
 id: FBbt:00058110
 name: larval testis
-def: "Immature testis of the larva. It contains only germ-line cells of pre-meiotic and meiotic stages until the primary spermatocyte stage (Gartner et al., 2014)." [FlyBase:FBrf0226408]
+def: "Immature testis of the larva. It contains only germ-line cells of pre-meiotic and meiotic stages until the primary spermatocyte stage (Gartner et al., 2014), as well as immature precursors of the adult testis epithelium (Mahadevaraju et al, 2021)." [FlyBase:FBrf0226408, FlyBase:FBrf0248088]
```

Comment:
perhaps move 'only' -> '...cells of only pre-meiotic...', otherwise this sounds a bit strange with your addition.